plugins/system-update/PageComponent.qml

```diff
@@ -172,6 +172,13 @@ ItemPage {
                 }
             }
 
+            ListItem.SingleValue {
+                objectName: "updateApps"
```

Comment:
Not needed

Reply:
Ah, I see it's only for testing. Interesting.